### DISCUSSION TITLE
Survey about developer communication preferences in GitHub projects

### DISCUSSION THREAD

**Discussion by verenya** [community/community#131775] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hello,

I am a PhD student at the university of Stuttgart, Germany. During the last few months, I have interviewed GitHub maintainers about communication in their projects. Now, I want to hear from you, the developers, to get your valuable insights!

For this, we are currently conducting a short (approx. 10 mins) survey. If you have experience as a GitHub developer, please consider participating here:

https://forms.gle/5HxjiUJeFRR8PoWn8

Your input will not only help me with my PhD, but also contribute to the broader knowledge in the developer community.

Thank you in advance for your time and valuable contributions.

**Comment by ebndev** [comment]
Hi @verenya, thanks for sharing your with the community! 

Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
